layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutController.kt

```diff
@@ -216,8 +217,12 @@ internal class FoldableLayoutController constructor(
      */
     private fun addHingeIfNeeded() {
         if (!rootView.hasChild(R.id.hinge_id)) {
-            foldingFeature?.generateHingeView(rootView.context)?.let { hingeView ->
-                rootView.addView(hingeView, 1)
+            // Some foldable emulators return the the foldable feature as 1 px thick (instead of 0px).
+            // In this case the Hinge View should no be added
+            if (foldingFeature.getHingeThickness() > 1) {
```

Comment:
I'm not sure if we want this. So here we are saying that if we have a hinge greater than 1px we will create a representation of it. If it's 0 we don't need it. I guess because when is 0 the start/end panes are just one after the other. I think we should follow the same logic no matter the hinge thickness.

The problem, originally, was that there is a hinge representation in black that shouldn't be there, so imho, if we need that representation because future calculations or panes adjustment, then make the hinge representation transparent.
This will solve the problem in all cases.

Because in the code above what we are doing is just if there is a hinge with 0px, don't create the representation, but in all other cases yes. I think we should have just one way to go and no speciifc cases for specific hinge-thickness.

Reply:
We don't add the hinge view if the device/emulator says the folding feature is 0px or 1px thick.
I am not sure we want to draw a transparent hinge view because the user may see the fragmet's /actvity's UI behind the Foldable layout (depending on how the UI is build).

Reply:
Update:
It seems that the HingeView is required by the NavigationComponent and it always needs to be there.
I've change the logic to make the HingeView 0xp thick instead of removing it.

Reply:
Andrei, that code was reverted. 
The getHingeThickness doesn't longer exists.